src/Compilers/Core/CodeAnalysisTest/EditorConfigTests.cs

```diff
@@ -27,7 +29,7 @@ public void SimpleCase()
 [*.cs]
 my_prop = my_val
 
-", "Z:\\bogus");
```

Comment:
Why the change?

Reply:
Ah, now I see your comment below about on `/` being recognized as a dir separator.

---
In reply to: [191015436](https://github.com/dotnet/roslyn/pull/27119#discussion_r191015436) [](ancestors = 191015436)